src/styles/_nav.scss

```
@@ -0,0 +1,39 @@
+.nav {
+  color: black;
+  text-transform: uppercase;
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+  }
+
+  &__item {
+    padding: 7px 20px;
+    margin: 0 22.5px;
+    border-radius: 10px;
+
+    &_active {
+      background-color: $active-color;
+    }
+  }
+
+  &__link {
+    color: white;
+    text-decoration: none;
+
+    transition: color $hover-duration;
+
+    &:hover {
+      color: $active-color;
+    }
+  }
+
+}
+
+@media screen and (max-width: $small-screen-breakpoint) {
+  .nav__list {
+    flex-direction: column;
+    align-items: center;
+  }
+}
```

```suggestion
    }
  }

  @media screen and (max-width: $small-screen-breakpoint) {
    &__list {
      flex-direction: column;
      align-items: center;
    }
  }
```
медиа запрос тоже можно вкладывать

---

done